BonusCalcApi/V1/Controllers/BandChangesController.cs

```diff
@@ -0,0 +1,75 @@
+using BonusCalcApi.V1.Boundary.Response;
+using BonusCalcApi.V1.Exceptions;
+using BonusCalcApi.V1.Factories;
+using BonusCalcApi.V1.Infrastructure;
+using BonusCalcApi.V1.UseCase.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BonusCalcApi.V1.Controllers
+{
+    [ApiController]
+    [Route("api/v1/changes")]
```

Comment:
It's a minor gripe but "changes" could mean changes to anything here - it's the first post-version element in the route. Can we use /bandchanges or /bands/changes

Reply:
How about `/band-changes` ?